compiler/tests/data/decimal-singleton2.bal

```diff
@@ -0,0 +1,14 @@
+// Zero<:DECIMAL
+// MinZero<:DECIMAL
+// One<:DECIMAL
+// FpOne<:DECIMAL
+// Zero<:MinZero
+// MinZero<:Zero
+// One<:FpOne
+// FpOne<:One
+
+type Zero 0d;
+type MinZero -0d;
```

Comment:
MinusZero (Min is abbreviation for minimum)

Reply:
fixed.